environments/example-credentials/services/scoringmp/overlays/bootstrap.properties

```diff
@@ -6,7 +6,7 @@ mp.messaging.connector.liberty-kafka.security.protocol=SASL_SSL
 mp.messaging.connector.liberty-kafka.ssl.protocol=TLSv1.2
 mp.messaging.connector.liberty-kafka.sasl.mechanism=PLAIN
 # Make sure you set the username and API key at the end
-mp.messaging.connector.liberty-kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="token" password="<API-KEY>";
+mp.messaging.connector.liberty-kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="token" password="<KAFKA_PASSWORD>";
```

Comment:
Should the username be passed through here too, or is this just a minimal change to keep scoringmp working for now?

Reply:
Since we are not using KAFKA APIKEY and instead reference that as KAFKA PASSWORD from now own to align with SCRAM I updated that there for reference.
We will need to look into por detail at scoring mp deployment options since back at the time the microprofile was not reading env variables properly and thus the reason for injecting the bootstrap.properties file (already tailored with the config needed) rather than defining env variables of the type MP_MESSAGING_XXX